[2023-08-22 15:25:47,072] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: new_etl.transformar_data manual__2023-08-22T15:25:41.037149+00:00 [queued]>
[2023-08-22 15:25:47,095] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: new_etl.transformar_data manual__2023-08-22T15:25:41.037149+00:00 [queued]>
[2023-08-22 15:25:47,097] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-08-22 15:25:47,098] {taskinstance.py:1377} INFO - Starting attempt 1 of 4
[2023-08-22 15:25:47,099] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-08-22 15:25:47,113] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): transformar_data> on 2023-08-22 15:25:41.037149+00:00
[2023-08-22 15:25:47,125] {standard_task_runner.py:52} INFO - Started process 14705 to run task
[2023-08-22 15:25:47,129] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'new_etl', 'transformar_data', 'manual__2023-08-22T15:25:41.037149+00:00', '--job-id', '195', '--raw', '--subdir', 'DAGS_FOLDER/etl2.py', '--cfg-path', '/tmp/tmpk7phazdl', '--error-file', '/tmp/tmpxmuux4id']
[2023-08-22 15:25:47,131] {standard_task_runner.py:80} INFO - Job 195: Subtask transformar_data
[2023-08-22 15:25:47,224] {task_command.py:371} INFO - Running <TaskInstance: new_etl.transformar_data manual__2023-08-22T15:25:41.037149+00:00 [running]> on host 0110d0e687e9
[2023-08-22 15:25:47,304] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Sol
AIRFLOW_CTX_DAG_ID=new_etl
AIRFLOW_CTX_TASK_ID=transformar_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-22T15:25:41.037149+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-22T15:25:41.037149+00:00
[2023-08-22 15:25:47,389] {python.py:173} INFO - Done. Returned value was: None
[2023-08-22 15:25:47,404] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=new_etl, task_id=transformar_data, execution_date=20230822T152541, start_date=20230822T152547, end_date=20230822T152547
[2023-08-22 15:25:47,424] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-22 15:25:47,458] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
